iocore/net/SSLUtils.cc

```diff
@@ -2052,6 +2052,7 @@ SSLParseCertificateConfiguration(const SSLConfigParams *params, SSLCertLookup *l
 
   // Load the global ticket key for later use.
   REC_ReadConfigStringAlloc(ticket_key_filename, "proxy.config.ssl.server.ticket_key.filename");
+  ticket_block_free(global_default_keyblock);
```

Comment:
Unfortunately this is not going to work correctly. At the same time you have frees the ticket block here, other threads are accessing it to perform SSL handshaking, so they will read a stale pointer.

One way to fix this is to make the ticket block part of the ``SSLCertLookup`` object and then point the global pointer that that object. That might get a little nasty since updating the certificate lookup can fail after you assign the global pointer. You might need to find a different way.

In general, I think what needs to happen here is:

- Only update the ticket block if it has changed. Updating is each time will reduce the possibility of SSL session resumption.
- If you update the ticket block, swap the pointers to the old and new blocks atomically, then schedule the old block for deletion